dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/query/Restrictions.java

```diff
@@ -28,21 +28,7 @@
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-import org.hisp.dhis.query.operators.BetweenOperator;
-import org.hisp.dhis.query.operators.EmptyOperator;
-import org.hisp.dhis.query.operators.EqualOperator;
-import org.hisp.dhis.query.operators.GreaterEqualOperator;
-import org.hisp.dhis.query.operators.GreaterThanOperator;
-import org.hisp.dhis.query.operators.InOperator;
-import org.hisp.dhis.query.operators.LessEqualOperator;
-import org.hisp.dhis.query.operators.LessThanOperator;
-import org.hisp.dhis.query.operators.LikeOperator;
-import org.hisp.dhis.query.operators.MatchMode;
-import org.hisp.dhis.query.operators.NotEqualOperator;
-import org.hisp.dhis.query.operators.NotInOperator;
-import org.hisp.dhis.query.operators.NotLikeOperator;
-import org.hisp.dhis.query.operators.NotNullOperator;
-import org.hisp.dhis.query.operators.NullOperator;
+import org.hisp.dhis.query.operators.*;
```

Comment:
We don't usually do import rollups.. maybe you need to adjust your IDE settings?